site2/docs/admin-api-topics.md

```diff
@@ -538,6 +538,31 @@ admin.topics().getMessageById(topic, ledgerId, entryId);
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+### Get message ID 
+
+You can get message id published at or just after the given datetime.
```

Comment:
Why the description here is different from the previous one (`Get message ID by timestamp`)?

Reply:
This doc is for user, it's more detailed. Modified the previous one in new commit. Please check it out.